modules/update-service-install-web-console.adoc

```diff
@@ -0,0 +1,34 @@
+[id="update-service-install-web-console_{context}"]
+= Installing the {product-title} Update Service by using the web console
```

Comment:
Same here. It's introducing the way to install the Update Service operator but the title is saying Installing the Update Service.

Reply:
Agree. I've introduced the sections headers with either `Operator` or `application` to distinguish between operator and operand. This PR is obsolete and will be closed. Please see https://github.com/openshift/openshift-docs/pull/29630 for latest.